in.go

```diff
@@ -109,9 +123,10 @@ func Get(request GetRequest, github Github, git Git, outputDir string) (*GetResp
 
 // GetParameters ...
 type GetParameters struct {
-	SkipDownload    bool   `json:"skip_download"`
-	IntegrationTool string `json:"integration_tool"`
-	GitDepth        int    `json:"git_depth"`
+	SkipDownload      bool   `json:"skip_download"`
+	IntegrationTool   string `json:"integration_tool"`
+	GitDepth          int    `json:"git_depth"`
+	ChangedFilesQuery bool   `json:"generate_changed_file_list"`
```

Comment:
`ChangedFilesQuery` should be `GenerateChangedFileList` to match whatever parameter name we go for (makes it easier to read the code).

Reply:
Sure